packages/cli/src/build/config.ts

```diff
@@ -57,6 +57,11 @@ function getGlobal(project: Project, name: string) {
   }
 }
 
+const babelHelperId = /@babel\/runtime(|-corejs[23])\/helpers\//;
+
+const interop = (id: string | null): "auto" | "default" =>
```

Comment:
TIL that this can be a function

Reply:
Have you checked how this will behave in Babel 8? I know that more than this would have to be re-checked to check against Babel 8 compatibility but maybe worth to be future-proof here. 

In Babel 8 there won't be separate cjs/esm versions of helpers as they have started to use conditional exports. You would have to check if `interop: "default"` works with Babel 8 though - there might have been changes around this (but they might have also left it as is - I don't recall, I think Nicolo has wanted to support `.default`-less requires in node so probably this has not been touched but... well, worth re-checking).

Reply:
Given [the PR about it](https://github.com/babel/babel/pull/12295) isn't merged yet, I'm not overly concerned about it. Preconstruct has a direct dependency on @babel/core@7 rn anyway.